draft-thomson-https-local-domains.md

```diff
@@ -532,29 +555,48 @@ connection is made to that address, those are also considered
 * fc00::/7 (from {{?RFC4193}})
 * 127/8 and ::1/128 (from {{?RFC990}} and {{?RFC4291}})
 
-# Incremental Deployment
+# Operational Considerations
+
+## Incremental Deployment
 
 Where a server's hostname can be configured, a motivated network
 administrator can configure server hostnames to comply with this
 specification to provide immediate value to supporting clients.
 
-# Security Considerations
-
-TODO: write more on security considerations
+## Server Identity Change
 
-Because the server's public key is encoded into its domain name,
-changing the public key would also change its domain name -- thus, its
+The server's public key is encoded into its domain name.
+Changing the public key would also change its domain name -- thus, its
 identity as known by client password managers and other configurations
-in clients (e.g., printer, SMB share, etc.).  As such an identity
-change is extremely disruptive, it needs to be avoided.  This means
-the public/private key pair on a server needs to stay static.  The
-tradeoff is servers are vulnerable to their private keys being stolen
-and an active attacker intercepting traffic to that server.  The
-alternatives are to continue using unencrypted communication to local
-servers, which is vulnerable to passive attack, or to condition users
-to validate self-signed certificates for local servers.
+in clients (e.g., printer, SMB share, etc.). As such an identity
+change is extremely disruptive, it needs to be avoided.
 
+## Simplify DNS
 
+To simplify access to devices within a household, administrator can configure
+an in-house DNS server and create short, user-friendly DNS names that resolve
+to longer, more complex names. Configure the in-house DNS server to map
+these short names to their corresponding long names. This approach
+ensures that users can easily connect to the desired device using
+a simpler, more memorable DNS name.
+
+Implementing DNSSEC {{?RFC4033}} and using DNS over HTTPS (DoH) RFC 8484
+or DNS over TLS (DoT) {{?RFC7858}} for DNS queries can enhance the
+reliability of this system.
```

Comment:
```suggestion
```

Nobody gunna run local DNS server; that's what Bonjour/DNS-SD solve. Nobody gunna run DNSSEC on a small network. Bonjour/DNS-SD don't have a good way to secure themselves, so I also would not bring them up.  So: delete this section.